docs/troubleshooting/README.md

```diff
@@ -583,29 +696,100 @@ At times, we might need more information in that case please take a look at the
         version: v1
 ```
 
-from looking at the **work status** and specifically the **manifestConditions** section we could see that the namespace could not be applied but the deployment within the namespace got propagated from hub to the member cluster correctly. In this case to solve this issue maybe delete the existing namespace on the member cluster but that's upto the user to decide since the namespace could already contain resources within it.
+From looking at the `Work` status and specifically the `manifestConditions` section, we could see that the namespace could not be applied but the deployment within the namespace got propagated from hub to the member cluster.
 
-### How and where to find the correct Work resource?
+#### Resolution:
+In this scenario, a potential solution is to delete the existing namespace on the member cluster. However, it's essential to note that this decision rests with the user, as the namespace might already contain resources.
+
+### How can I debug when some clusters are not selected as expected?
+
+Check the status of the `ClusterSchedulingPolicySnapshot` to determine which clusters were selected along with the reason.
 
-We need to have the member cluster's namespace **fleet-member-{clusterName}**, ClusterResourceBinding's name **{CRBName}** and ClusterResourcePlacement's name **{CRPName}**.
+### How can I debug when a selected cluster does not have the expected resources on it or if CRP doesn't pick up the latest changes?
+
+Please check the following cases,
+- Check to see if `ClusterResourcePlacementSynchronized` condition in CRP status is set to `true` or `false`.
+- If it's set to `false` check this [question](#how-can-i-debug-when-my-crp-status-is-clusterresourceplacementsynchronized-condition-status-is-set-to--false--).
+- If it's set to `true`,
+  - Check to see if `ClusterResourcePlacementApplied` condition is set to `unknown`, `false` or `true`.
+  - If it's set to `unknown`, please wait as the resources are still being applied to the member cluster (if it's stuck in unknown state for a while, please raise a github issue as it's an unexpected behavior).
+  - If it's set to `false`, check this [question](#how-can-i-debug-when-my-crp-clusterresourceplacementapplied-condition-is-set-to--false--).
+  - If it's set to `true`, check to see if the resource exists on the hub cluster. The `ClusterResourcePlacementApplied` condition is set to `true` if the resource doesn't exist on the hub.
```

Comment:
- If it's set to `true`, check to see if the resource exists on the hub cluster.~~The `ClusterResourcePlacementApplied` condition is set to `true` if the resource doesn't exist on the hub.~~


The second statement looks confusing.